subsys/bluetooth/controller/ticker/ticker.h

```diff
@@ -229,15 +229,24 @@ uint8_t ticker_priority_set(uint8_t instance_index, uint8_t user_id,
 #if defined(CONFIG_BT_TICKER_EXT)
 struct ticker_ext {
 #if !defined(CONFIG_BT_TICKER_SLOT_AGNOSTIC)
-	uint32_t ticks_slot_window;/* Window in which the slot
-				    * reservation may be re-scheduled
-				    * to avoid collision
-				    */
-	uint32_t ticks_drift;      /* Actual drift since last expiry */
-	uint8_t reschedule_state;  /* State of re-scheduling of the
-				    * node. See defines
-				    * TICKER_RESCHEDULE_STATE_XXX
-				    */
+	uint32_t ticks_slot_window;    /* Window in which the slot
+					* reservation may be re-scheduled
+					* to avoid collision
+					*/
+	uint32_t ticks_drift;          /* Actual drift since last expiry,
+					* includes any ticker update interface
+					* made changes plus drift due to
+					* reschedule when not
+					* is_jitter_in_window, otherwise is only
+					* the value of drift due to reschedule
+					*/
+	uint8_t reschedule_state:3;    /* State of re-scheduling of the
+					* node. See defines
+					* TICKER_RESCHEDULE_STATE_XXX
+					*/
```

Comment:
If a new value is added to `TICKER_RESCHEDULE_STATE_` then this will silently start failing, right? 

How can we prevent a new `TICKER_RESCHEDULE_STATE_` value to fail silently? Add comment to the `TICKER_RESCHEDULE_STATE_` defines? BUILD_ASSERT?

Reply:
BUILD_ASSERT will be best... will investigate on how to add the check.

Reply:
I think we already get a compiler warning if the number of states becomes larger than what we can store in `reschedule_state`